### DISCUSSION TITLE
unable to install brew on macbook air m1

### DISCUSSION THREAD

**Discussion by cehyash** [Homebrew/discussions#5011] [Getting started]
### Output of `brew config`

```shell
curl: (28) Failed to connect to raw.githubusercontent.com port 443 after 75226 ms: Couldn't connect to server
```

### Output of `brew doctor`

```shell
curl: (28) Failed to connect to raw.githubusercontent.com port 443 after 75226 ms: Couldn't connect to server
```

### Description of issue

whenever i paste this installation command on my terminal: /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
it always throws this error : curl: (28) Failed to connect to raw.githubusercontent.com port 443 after 75226 ms: Couldn't connect to server 
<img width="1080" alt="Screenshot 2023-12-27 at 2 00 48 PM" src="https://github.com/Homebrew/discussions/assets/74892388/88215857-485a-4a98-9ec9-35e5dd620a5d">

**Comment by SMillerDev** [comment]
If you can't connect to GitHub you should check and improve your internet connection.

**Reply by cehyash** [reply]
umm, i have checked and there's nothing wrong with that as well 
<img width="1153" alt="Screenshot 2023-12-27 at 3 05 46 PM" src="https://github.com/Homebrew/discussions/assets/74892388/ca274c12-958e-405a-a4b5-9a07beacaaa0">

**Reply by SMillerDev** [reply]
It might be fast, but it's still broken because it can't connect to GitHub.
And there is nothing Homebrew can do there

**Reply by cehyash** [reply]
ohh alright, appreciate it. 
Thanks :)
